jobs_worker.go

```diff
@@ -69,20 +68,11 @@ func (w *JobsWorker) Start(opts *JobsWorkerOptions) {
 }
 
 func (w *JobsWorker) ServiceFunc(ctx context.Context) error {
-	redisAddress, err := ExtractHostAndPort(w.Config.JobsEnqueuer.URL)
+	redisPool, err := BuildRedisPool(w.Config.JobsEnqueuer.URL, w.Options.Concurrency)
 	if err != nil {
 		return fmt.Errorf("failed to extract host and port from redis URL: %w", err)
```

Comment:
I believe, we should also change the error message here.

Reply:
Fixed